docs/en/Xamarin.F50/Preferences.xml

```diff
@@ -0,0 +1,81 @@
+<Type Name="Preferences" FullName="Xamarin.F50.Preferences">
+  <TypeSignature Language="C#" Value="public class Preferences" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit Preferences extends System.Object" />
+  <AssemblyInfo>
+    <AssemblyName>Xamarin.F50</AssemblyName>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>To be added.</summary>
+    <remarks>To be added.</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public Preferences ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>To be added.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public Preferences (Android.Content.Context context);" />
```

Comment:
NOTE: the platform-specific member lives here.

Reply:
So mdoc has no specific way of associating docs with platforms?

Reply:
The docs just describe the API, in a platform-independent way. We could create separate docs for each assembly, but I wanted to add them all together so that the one docs can have everything in one place. Of course, having this item in the docs doesn't mean that it will appear in the intellisense as this member does not exists.